x-pack/plugins/uptime/public/components/overview/monitor_list/columns/monitor_name_col.tsx

```diff
@@ -41,10 +41,12 @@ export const MonitorNameColumn = ({ summary }: Props) => {
   const linkParameters = stringifyUrlParams(params, true);
 
   const currFilters = parseCurrentFilters(params.filters);
+  const currExcludedFilters = parseCurrentFilters(params.excludedFilters);
 
   const [filterType, setFilterType] = useState<string[]>(currFilters.get('monitor.type') ?? []);
+  const excludedFilters = React.useRef<string[]>(currExcludedFilters.get('tags') ?? []);
```

Comment:
`currExcludedFilters.get('tags')` would be `currExcludedFilters.get('monitor.type')`

Reply:
Yeah, surprisingly the smoke test continued to work even with this field mismatch. I have pushed https://github.com/elastic/kibana/pull/105739/commits/a05766f9e0c9514669317ded0659798acd283e32.